include/nrsc5.h

```diff
@@ -25,6 +25,11 @@
 #define NRSC5_MIME_TEXT             0xBB492AAC
 #define NRSC5_MIME_JPEG             0x1E653E9C
 #define NRSC5_MIME_PNG              0x4F328CA0
+#define TTN_TPEG_1                  0xB39EBEB2
```

Comment:
Should we put a ```NRSC5_``` prefix on these for namespacing?

Reply:
Yes; I forgot to do that when i copied them over from Python.